monai/networks/utils.py

```diff
@@ -693,7 +695,7 @@ def convert_to_onnx(
             f = io.BytesIO()
```

Comment:
Could you please also modify this part based on the latest api from `torch.onnx.export`? Thanks!
https://github.com/Project-MONAI/MONAI/issues/8149#issuecomment-2422165354